packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -186,5 +186,43 @@ placement. Supported values are `top`, `bottom`, `left`, and `right`.
 👉 Please consult [Floating UI][floating-ui] documentation to understand how it
 works and to get an idea of all possible cases you may need to cover.
 
+## Javascript API
+
+### Methods
+
+| Method                | Description                                                                                                                                                                                                                                                      |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `getInstance`         | _Static_ method which allows you to get the tooltip instance associated with a DOM element, or create a new one in case it wasn’t initialized                                                                                                                    |
+| `getOrCreateInstance` | _Static_ method which allows you to get the tooltip instance associated with a DOM element, or create a new one in case it wasn’t initialized                                                                                                                    |
```

Comment:
```suggestion
| `getOrCreateInstance` | _Static_ method which allows you to get the tooltip instance associated with a DOM element, or create a new one in case it wasn’t initialized.                                                                                                                    |
```

Reply:
Fixed.